### ISSUE TITLE
Bot sends no mumble reply if text message is too long.

### ISSUE THREAD

**Issue by BafDyce** [BafDyce/botamusique#43]
E.g. running `!list` with a huge list of files in the music directory leads to a long reply message. Depending on the mumble server's settings this message might be rejected by the server.

The bot should be able to determine whether a given message was successfully sent (or determine the maximimum message length in characters). Then the message should be either truncated, split into multiple messages, or it should just send an error message.

**Comment by azlux** [comment]
The important part is the server. If the client don't support it, you will have a message that the text is too long.

**Comment by BafDyce** [comment]
Yeah, I know. I'm already working on the client side of this feature.
